pkg/api/http_server.go

```diff
@@ -659,6 +664,61 @@ func (hs *HTTPServer) apiHealthHandler(ctx *web.Context) {
 	}
 }
 
+// setupFrontendLogHandlers will set up handlers for logs incoming from frontend.
+// handlers are setup even if frontend logging is disabled, but in this case do nothing
+// this is to avoid reporting errors in case config was changes but there are browser
+// sessions still open with older config
+func (hs *HTTPServer) setupFrontendLogHandlers() {
```

Comment:
Nit. Could consider moving this to frontend_logging.go.

Reply:
- [ ] move to frontend_logging.go